packages/opensrp-plans/src/ducks/index.ts

```diff
@@ -0,0 +1,340 @@
+import { Dictionary } from '@onaio/utils';
+import intersect from 'fast_array_intersect';
+import { get, keyBy, values } from 'lodash';
+import { AnyAction, Store } from 'redux';
+import { createSelector } from 'reselect';
+import SeamlessImmutable from 'seamless-immutable';
+import { PlanDefinition } from '../plan-global-types';
+import { descendingOrderSort, isPlanDefinitionOfType } from '../helpers/utils';
+
+/** the reducer name */
+export const reducerName = 'PlanDefinition';
+
+/** list of plan types displayed
+ * TODO: To be moved to env
+ */
+export const DISPLAYED_PLAN_TYPES = 'FI,IRS,MDA,MDA-Point,Dynamic-FI,Dynamic-IRS,Dynamic-MDA';
+
+// actions
+
+/** PLAN_DEFINITION_FETCHED action type */
+export const PLAN_DEFINITIONS_FETCHED =
+  'reveal/reducer/opensrp/PlanDefinition/PLAN_DEFINITIONS_FETCHED';
+
+/** PLAN_DEFINITION_FETCHED action type */
+export const REMOVE_PLAN_DEFINITIONS =
+  'reveal/reducer/opensrp/PlanDefinition/REMOVE_PLAN_DEFINITIONS';
+
+/** PLAN_DEFINITION_FETCHED action type */
+export const ADD_PLAN_DEFINITION = 'reveal/reducer/opensrp/PlanDefinition/ADD_PLAN_DEFINITION';
+
+/** interface for plan Objects */
+/** Enum representing the possible intervention types */
+export enum PlanStatus {
+  ACTIVE = 'active',
+  COMPLETE = 'complete',
+  DRAFT = 'draft',
+  RETIRED = 'retired',
+}
+/** Enum representing the possible intervention types */
+export enum InterventionType {
```

Comment:
Can we add the EUSM plan type?

Reply:
@moshthepitt Seems Eusm doen't have plan type 
https://github.com/OpenSRP/web/issues/180#issuecomment-732754999 @dubdabasoduba can confirm this

Reply:
@p-netm @kahummer @dubdabasoduba any updates on this?

Reply:
@kahummer [see here](https://opensrp.slack.com/archives/G01BMTANDLK/p1606833290004300?thread_ts=1606824414.002300&cid=G01BMTANDLK)

Reply:
@moshthepitt  @dubdabasoduba confirmed eusm plans don't have plan types

Reply:
@kahummer please check out the link shared by Peter M 5 hours ago.  We will use that plan type.